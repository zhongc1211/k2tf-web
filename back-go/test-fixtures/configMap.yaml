apiVersion: v1
kind: ConfigMap
metadata:
  name: foo-config-map
  namespace: bar
  labels:
    lbl1: somevalue
    lbl2: another
data:
  item1: wow
  item2: wee
