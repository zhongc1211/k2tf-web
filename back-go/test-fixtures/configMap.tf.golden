resource "kubernetes_config_map" "foo_config_map" {
  metadata {
    name      = "foo-config-map"
    namespace = "bar"
    labels    = { lbl1 = "somevalue", lbl2 = "another" }
  }
  data = { item1 = "wow", item2 = "wee" }
}
